### PR TITLE
Add bound to streamly version

### PR DESCRIPTION
**Pull request by rnjtranjan**

### CODE REVIEW

**Comment by harendra-kumar** [comment]
The title of the PR and commit contents do not seem to match.

**Comment by harendra-kumar** [comment]
Can you make two separate commits or two separate PRs?

File: streamly-statistics.cabal [hunk 1]
```diff
@@ -98,7 +98,7 @@ library
     import: ghc-options
     exposed-modules:     Streamly.Statistics
     build-depends:       base     >= 4.9 && < 5
-                       , streamly == 0.8.1.1
+                       , streamly >= 0.8.1 && < 0.9
```
Comment (harendra-kumar) [inline streamly-statistics.cabal:101]: You can rebase and remove these changes.

File: src/Streamly/Statistics.hs [hunk 2]
```diff
@@ -649,7 +655,7 @@ ewma k = extract <$> Fold.foldl' step (Tuple' 0 1)
 {-# INLINE ewmaAfterMean #-}
 ewmaAfterMean :: Monad m => Int -> Double -> Fold m Double Double
 ewmaAfterMean n k =
-    Fold.concatMap (\i -> (Fold.foldl' (ewmaStep k) i)) (Fold.take n Fold.mean)
+    Fold.concatMap (Fold.foldl' (ewmaStep k))  (Fold.take n Fold.mean)
```
Comment (harendra-kumar) [inline src/Streamly/Statistics.hs:658]: Can you put these refactoring changes in a separate commit?

File: src/Streamly/Statistics.hs [hunk 3]
```diff
@@ -862,3 +868,21 @@ sampleStdDev = sqrt <$> sampleVariance
 {-# INLINE stdErrMean #-}
 stdErrMean :: (Monad m, Floating a) => Fold m (a, Maybe a) a
 stdErrMean = Fold.teeWith (\sd n -> sd / sqrt n) sampleStdDev length
+
+bubbleAsc :: (MonadIO m, Prim a) =>
+    Int -> (a -> a -> Ordering) -> Array.Array a -> m ()
+bubbleAsc idx cmp arr = do
```
Comment (harendra-kumar) [inline src/Streamly/Statistics.hs:874]: We can add this in streamly mutable array module itself, in the mutable algorithms section.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]

**Comment by rnjtranjan** [comment]
new PR is created : https://github.com/composewell/streamly/pull/1570

**Pull request closed without merging**